main.py

```diff
@@ -63,9 +63,14 @@ def change_read():
 
     book_id = ui.get_book_id()
     book = store.get_book_by_id(book_id)  
-    new_read = ui.get_read_value()     
-    book.read = new_read 
+    new_read = ui.get_read_value()  
+    book.read = new_read    
     book.save()
+    # ui.get_read_value() returns a True value if the user responds 'read' and False for 'not-read' 
```

Comment:
Remove unused code

Reply:
@claraj are you saying don't put comments in the code (and put them instead in the Github conversations)?

Reply:
Do add comments to code!
Remove unused code, don't comment out unused code.